src/resources/postcss/tickets-commerce/admin/orders/single.pcss

```diff
@@ -0,0 +1,286 @@
+/**
+ * Event Tickets - Single Admin Order Page
+ *
+ * @since TBD
+ */
+
+body.wp-admin.post-type-tec_tc_order {
+	@media screen and (max-width: 782px) {
+		.tribe-mobile-hidden {
+			display: none;
+		}
+	}
+
+	@media screen and (max-width: 1100px) {
+		.tribe-desktop-only {
+			display: none;
+		}
+	}
+
+	.tribe-mobile-only {
+		display: none;
+
+		@media screen and (max-width: 782px) {
+			display: inline-block;
+		}
+	}
+
+	input#title,
+	#post-body-content,
+	.dataTables_info,
+	#minor-publishing {
+		display: none;
+	}
+
+	#tribe-tickets-order-details {
+		.postbox-header {
+			display: none;
+		}
+
+		h2 {
+			color: #172b4d;
+			font-size: var(--tec-font-size-5);
+			font-weight: var(--tec-font-weight-regular);
+			line-height: var(--tec-line-height-2);
+			margin-left: var(--tec-spacer-1);
+			padding: 10px 0;
+		}
+	}
+
+	#tribe-tickets-order-items {
+		.inside {
+			margin-top: 0;
+			padding: 0;
+		}
+
+		.postbox-header {
+			border-bottom: none;
+		}
+
+		.handle-actions {
+			display: none;
+		}
+	}
+
+	.tribe-tickets-commerce-extend-order-row {
+		background: none;
+		border: 0;
+		color: #787c82;
+		cursor: pointer;
+		height: 1.62rem;
+		margin: 0;
+		padding: 0;
+		width: 1.62rem;
+
+		.tribe-tickets-commerce-order-extend-indicator {
+			&:before {
+				color: #3c434a;
+				content: "\f347";
+				display: inline-block;
+				font-family: dashicons; /* stylelint-disable-line */
+				font-size: 20px;
+				font-style: normal;
+				height: 20px;
+				line-height: 1;
+				position: relative;
+				text-decoration: none;
+				top: 0.11rem;
+				transition: var(--tec-transition-transform);
+				width: 20px;
+			}
+		}
+	}
+}
+
+.tec-tickets-commerce-single-order {
```

Comment:
While I typically don't have an issue with `&--` in postcss your nesting is getting deep.  Especially in items, below.

Tracking once you are 100+ lines down the file can get confusing. Might be best to break these up.

Would also probably eliminate the need for the `/* stylelint-disable-line */`?